fe1-web/src/features/digital-cash/reducer/DigitalCashReducer.ts

```diff
@@ -45,6 +51,20 @@ const initialState: DigitalCashLaoReducerState = {
 /* Name of digital cash slice in storage */
 export const DIGITAL_CASH_REDUCER_PATH = 'digitalCash';
 
+const createEntryAndAddInvPubHash = (
```

Comment:
not sure if the `Inv` was left here intentionally after renaming all other occurrences? :)

Reply:
No it's definitely not intentional, thanks! 😅